src/main/java/org/jitsi/videobridge/influxdb/LoggingHandler.java

```diff
@@ -86,27 +86,27 @@
             "conference_id",
             "endpoint_id",
 
-            "stats.local_ip",
-            "stats.local_port",
-            "stats.remote_ip",
-            "stats.remote_port",
+            "stats_local_ip",
```

Comment:
0.9.2 has a bug where it doesn't accept column names that start with reserved words. (https://github.com/influxdb/influxdb/issues/3547)